drivers/serial/uart_mcux_lpuart.c

```diff
@@ -356,7 +356,11 @@ static int mcux_lpuart_init(const struct device *dev)
 	uart_api_config->stop_bits = UART_CFG_STOP_BITS_1;
 	uart_api_config->data_bits = UART_CFG_DATA_BITS_8;
 	uart_api_config->flow_ctrl = UART_CFG_FLOW_CTRL_NONE;
-
+#if defined (CONFIG_BT_UART_ON_DEV_NAME)
+	if (strcmp(CONFIG_BT_UART_ON_DEV_NAME, dev->name) == 0) {
+		uart_api_config->flow_ctrl = UART_CFG_FLOW_CTRL_RTS_CTS;
```

Comment:
Flow control should be implemented using the dts property, `hw-flow-control`, similar to `drivers/serial/uart_mcux.c`. We should not use `CONFIG_BT_UART_ON_DEV_NAME` here.

Reply:
Thanks @MaureenHelm , I have upated it, please help check it again.